theme/mixins/backdrop.less

```diff
@@ -7,4 +7,5 @@
   background-color: @sk-black;
   opacity: 0.7;
   transition: opacity 0.3s;
+  z-index: @z-index-negative;
```

Comment:
Tried it doesn't work. Backdrop is still kinda work in progress, and you can opt-in when using autosuggest, so it's better to have it for now, till we have access to isOpen state of autosuggest then we could twick z-indexes based on it.